lib/mayaUsd/ufe/UsdPointInstanceModifierBase.h

```diff
@@ -275,8 +342,27 @@ class MAYAUSD_CORE_PUBLIC UsdPointInstanceModifierBase
         return usdAttr;
     }
 
+    void _closeBatch()
+    {
+        // To close the batch we simply remove it from the map of batches under
+        // construction, and thus it can no longer be joined.  If a batch was
+        // created, erase() will harmlessly fail for all modifiers except the
+        // first one.  If no batch was created, make a trivial, unshared batch
+        // just for this modifier.
+        batches().erase(pointInstancerPath());
+        if (!_batch) {
+            _batch = std::make_shared<UsdPointInstanceBatch<UsdValueType>>();
+            _batch->nbInstances = 1;
+        }
+    }
+
+    // An alternative to storing these three data members would be to simply
+    // store a UsdSceneItem.  PPT, 12-Apr-2021.
     PXR_NS::UsdPrim _prim;
     int             _instanceIndex;
+    Ufe::Path       _path {};
```

Comment:
There shouldn't be any need to initialize the path explicitly. BUT it is a good practice for trivial types like int (just above).